dvc/cli.py

```diff
@@ -2,6 +2,7 @@
 import argparse
 import logging
 import sys
+from gettext import gettext as _
```

Comment:
Not a fan of the `gettext` documentation.

```suggestion
from gettext import gettext
```

However since it's also not used in the rest of the codebase it's propbably best to leave out altogether and open a different issue for it.

```suggestion
```